lib/screen/interactive_postcard/postcard_detail_page.dart

```diff
@@ -654,7 +728,7 @@ class ClaimedPostcardDetailPageState extends State<ClaimedPostcardDetailPage>
               onTap: () async {
```

Comment:
i think we can use await here.
It's better to use await rather than unawaited so we don't have to use .then when we need return value.
If you don't want to do that now, could you remove async